MAUI/Shimmer/built-in-views.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the Built-in types support of Syncfusion .NET MAUI Shimmer (SfShimmer) control and more.
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Built-in Views of .NET MAUI Shimmer (SfShimmer)
+
+## Built-in types
+
+Syncfusion .NET MAUI Shimmer (SfShimmer) control provides seven different built-in shimmer view types.
```

Comment:
Syncfusion .NET MAUI Shimmer (SfShimmer) control provides seven built-in shimmer view types.

Reply:
Contents modified